src/experimental/shader-modules/index.js

```diff
@@ -1,16 +1,15 @@
-import {registerShaderModules} from '../shader-tools';
+// import * as fp64 from './fp64';
 
-import * as fp64 from './fp64';
-
-import * as fog from './fog';
-import * as picking from './picking';
+// import * as fog from './fog';
+// import * as picking from './picking';
 // import * as lighting from './lighting';
-import * as material from './material';
+// import * as material from './material';
 
-registerShaderModules([
-  fp64,
-  fog,
-  picking,
-  // lighting,
-  material
-]);
+// TODO - This probably defeats tree-shaking
```

Comment:
> Can we register the modules one by one in their own file so that only importing them will trigger the register?

The problem is that to support tree shaking we need double dist folders - one for non-tree-shaking aware bundlers, and one with imports/export left untranspiled for tree-shaking bundlers.

In this case, keeping multiple top level import files is complicated, we can't just put a file in the root that imports './dist/...' since dist might be different depending on bundler.

A good solution will be to just export a function registerPackages that the app needs to call. As long as we don't register them globally we are good.